args.go

```diff
@@ -115,6 +116,17 @@ func (a *Args) QueryString() []byte {
 	return a.buf
 }
 
+// Sort change args order which ASCII sorted by key, value.
```

Comment:
Can you please change this to:
```go
// Sort sorts Args lexicographically by key and then value.
```

Reply:
Args.args is not public , so maybe can't have the sort function.
I need time to think about it.